### DISCUSSION TITLE
How to find image which build by nerdctl or buildctl?

### DISCUSSION THREAD

**Discussion by jiangxiaobin96** [containerd/nerdctl#2859] [Q&A] [answered]
- nerdctl build -n k8s.io ...
After I build image by `nerdctl build -n k8s.io. ...`, i cannot find this image in `ctr -n k8s.io i ls`. `nerdctl image ls` is OK. But I want to move image to containerd which namespace is k8s.io.
- buildctl build
buildctl is the same question. After building image by `buildctl build`, i only find image in namespace `buildkit`, but how to move image to k8s.io.

**Comment by AkihiroSuda** [comment]
You may use `nerdctl save` and `nerdctl load` (or `ctr images export` and `ctr images import`) to move images across the namespaces

**Reply by jiangxiaobin96** [reply]
yes, I used  `ctr images export` to  tar file and `ctr images` from this tar file to k8s.io namespace, it seems too complicated. Is there any solutions to build image to k8s.io namespace directly?

**Comment by jiangxiaobin96** [comment, marked as answer]
`nerdctl -n k8s.io build -t ... ` can solve.
